### PR TITLE
update: translate the juicefs-setup.md to en & indicate we have supply the offical mul-platforms images

### PR DESCRIPTION
**Pull request by wang-mask**
### Ⅰ. Describe what this PR does
update: translate the juicefs-setup.md to en & indicate we have supply the offical mul-platforms images

### Ⅴ. Special notes for reviews

### CODE REVIEW

**Comment by cheyang** [comment]
@odidev Could you help review this docs? Thanks.

**Comment by odidev** [comment]
I followed this commit to setup Redis and minIO, and then proceeded to create a secret, following this doc: < https://github.com/fluid-cloudnative/fluid/blob/master/docs/en/samples/arm64.md >. 

Dataset successfully gets bound to runtime at first, as shown below: 
~~~ 
$ kubectl get dataset jfsdemo 

NAME      UFS TOTAL SIZE   CACHED   CACHE CAPACITY   CACHED PERCENTAGE   PHASE   AGE 
jfsdemo   [Calculating]    0.00B    4.00GiB          0.0%                Bound   95s 
~~~ 

However, the status changes to failed in some time. 
~~~ 
NAME      UFS TOTAL SIZE   CACHED   CACHE CAPACITY   CACHED PERCENTAGE   PHASE    AGE 
jfsdemo   0.00B            0.00B    4.00GiB          0.0%                Failed   17m 
~~~ 

Therefore, pods are in pending or container creating state: 
~~~ 
demo-app                 0/1     ContainerCreating   0          15m 
jfsdemo-fuse-pkvgf       0/1     Pending             0          15m 
jfsdemo-worker-0         1/1     Running             0          16m 
~~~

**Comment by cheyang** [comment]
@odidev Would you mind creating an issue so that we can track this in a separate thread?

**Comment by odidev** [comment]
Hi @cheyang , kindly refer https://github.com/fluid-cloudnative/fluid/issues/2155

File: docs/en/samples/arm64.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## Prerequisites
 
 ### Get the latest code of Fluid
-
+We currently provide the official image of Fluid for multiple platforms. You can refer to the [document](../userguide/get_started.md) to deploy Fluid diretly and jump to the step of *Example*.
```
Comment (cheyang) [inline docs/en/samples/arm64.md:6]: The official image for  multiple platforms is provided by Fluid community.

File: docs/en/samples/juicefs_setup.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
+# Steps to build Juicefs open source environment
+
+In order to quickly start Fluid + JuiceFS, you can quickly build an open source version of JuiceFS environment in Kubernetes by following the steps below. This environment is only used for functional verification, without any tuning, and is not recommended for production environments.
```
Comment (cheyang) [inline docs/en/samples/juicefs_setup.md:3]: In order to get started with Fluid + JuiceFS, you can quickly build community version of JuiceFS environment in Kubernetes by following the steps below.

File: docs/en/samples/juicefs_setup.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+The following example uses minIO as the back-end object storage and Redis as the metadata service.
```
Comment (cheyang) [inline docs/en/samples/juicefs_setup.md:5]: minIO -> MinIO

**Comment by cheyang** [comment]
> Hi @cheyang , kindly refer #2155

Thanks. Let's track it in the issue #2155

File: docs/en/samples/arm64.md (hunk 1 above)
Comment (wang-mask) [reply docs/en/samples/arm64.md]: fixed in https://github.com/fluid-cloudnative/fluid/pull/2154/commits/cb6304ed170a5f5254bd193223c7d4708e8d4d9f

**Pull request merged**